src/main/scala/Game.scala

```diff
@@ -0,0 +1,215 @@
+          cast = readInt()
+          cast match{
+            case 1 =>
+              if(mmslots <= 0){
+                println("You attempt to case this spell but you're out of Magic Missle juice!")
+              }
+              else{
+                mmslots = mmslots - 1
+                println("You cast Magic Missle!")
+                hit = rng(8, 1)
+                enemylife = enemylife - hit
+                println(s"You hit the enemy for $hit, current enemy life total is $enemylife")
+                if(enemylife <= 0){
+                  combat = 0
+                }
+              }
+            case 2 =>
+              if(fbslots <= 0){
+                println("You attempt to case this spell but you're out of Fire Bolt juice!")
+              }
+              else{
+                fbslots = fbslots - 1
+                println("You cast Fire Bolt!")
+                hit = rng(15, 1)
+                enemylife = enemylife - hit
+                println(s"You hit the enemy for $hit, current enemy life total is $enemylife")
+                if(enemylife <= 0){
+                  combat = 0
+                }
+              }
+            case 3 =>
+              if(tbslots <= 0){
+                println("You attempt to case this spell but you're out of Thunder Bolt juice!")
+              }
+              else{
+                tbslots = tbslots - 1
+                println("You cast Thunder Bolt!")
+                hit = rng(5, 3)
+                enemylife = enemylife - hit
+                println(s"You hit the enemy for $hit, current enemy life total is $enemylife")
+                if(enemylife <= 0){
+                  combat = 0
+                }
+              }
+            case 4 =>
+              if(rtslots <= 0){
+                println("You attempt to case this spell but you're out of Rock Throw juice!")
+              }
+              else{
+                rtslots = rtslots - 1
+                println("You cast Rock Throw!")
+                hit = rng(10, 5)
+                enemylife = enemylife - hit
+                println(s"You hit the enemy for $hit, current enemy life total is $enemylife")
+                if(enemylife <= 0){
+                  combat = 0
+                }
+              }
+            case 5 =>println("You do nothing. Are you trying to get yourself killed?")
+          }
+        }
+      case 5 =>println("You do nothing. Are you trying to get yourself killed?")
+    }
+  }
+
+  //
+  var cast = 0
+  var heal = 0
+  var choice = 0
+  var playeri = pinitiative
+  var enemyi = einitiative
+  var playerlife = 20
+  var enemylife = 20
+  var combat = 0
+  var pstart = 0
+  var estart = 0
+  var pac = 13
+  var eac = 13
+  var hit = 0
+  var mmslots = 2
+  var fbslots = 3
+  var tbslots = 1
+  var rtslots = 4
+  var instance1 = 0
+  var i = 0
+  var a_find = "open"
+  var b_find = "door"
+  var interact1 = 0
+  var interact2 = 0
+  //
+
+  while(instance1 == 0){
```

Comment:
similarly too big of a method

var msg= readLine() not reassigning value to it